google/cloud/bigtable/internal/defaults.cc

```diff
@@ -135,6 +137,21 @@ Options DefaultOptions(Options opts) {
     }
   }
 
+  auto const direct_path =
+      google::cloud::internal::GetEnv("GOOGLE_CLOUD_ENABLE_DIRECT_PATH")
+          .value_or("");
+  if (absl::c_any_of(absl::StrSplit(direct_path, ','),
+                     [](absl::string_view v) { return v == "bigtable"; })) {
+    opts.set<DataEndpointOption>(
+            "google-c2p:///directpath-bigtable.googleapis.com")
+        .set<AuthorityOption>("directpath-bigtable.googleapis.com");
+
+    // When using DirectPath the gRPC library already does load balancing across
+    // multiple sockets, it makes little sense to perform additional load
+    // balancing in the client library.
+    if (!opts.has<GrpcNumChannelsOption>()) opts.set<GrpcNumChannelsOption>(1);
```

Comment:
The ideal number of channels may not be `1`.  The service controls how many subchannels are created.  AFAIK, this is not exposed anywhere, but may be documented somewhere.  Consider a future PR to update this to `ceil($ncores / $subchannels)`

Reply:
Ack. I couldn't find the number. I will ask around.